source/_components/climate.intesishome.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: "IntesisHome AC Controller"
+description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
+date: 2019-03-12 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: intesishome.png
+ha_category: Climate
+ha_release: 
+ha_iot_class: "Cloud Push"
+---
+
+The `IntesisHome` climate platform let you control [IntesisHome](https://www.intesishome.com) devices via the IntesisHome cloud. IntesisHome provide integrations with air conditioners including including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
+
+Note: IntesisHome products are a separate product line to IntesisBox. This platform does not support IntesisBox devices (which can be controlled locally using the WMP protocol).
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: intesishome
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+```
+
+{% configuration %}
+username:
+  description: Your username for IntesisHome
+  required: true
+  type: string
+password:
+  description: Your password for IntesisHome
+  required: true
+  type: string
+{% endconfiguration %}
+
+- **username** (*Required*): Username of an account authorised to control your IntesisHome device on [IntesisHome.com](https://user.intesishome.com).
```

Comment:
Not sure why this is here? Please use the `description` of the above documentation configuration tags.